Libplanet.Net/SwarmOptions.cs

```diff
@@ -118,7 +93,8 @@ public enum TransportType : byte
         /// <summary>
         /// The type of <see cref="ITransport"/> used in <see cref="Swarm{T}"/>.
         /// </summary>
-        public TransportType Type { get; set; } = TransportType.TcpTransport;
+        public InitTransport.TransportType Type { get; set; }
```

Comment:
I think this change makes user import `InitTransport` class when creating `SwarmOptions` with option. Is it intended?